core-bundle/src/Command/Backup/BackupRestoreCommand.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Command\Backup;
+
+use Contao\CoreBundle\Doctrine\Backup\BackupManagerException;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+/**
+ * @internal
+ */
+class BackupRestoreCommand extends AbstractBackupCommand
+{
+    protected static $defaultName = 'contao:backup:restore';
+
+    protected function configure(): void
+    {
+        parent::configure();
+
+        $this
+            ->addOption('force', null, InputOption::VALUE_NONE, 'By default, this command checks whether the backup has been generated with Contao too. Use --force to disable this check.')
+            ->setDescription('Restores a backup.')
```

Comment:
any reason this does not support JSON as well?

Reply:
Because at the moment it does not provide any more info other than "worked" or "failed" which the status code already provides.